spec/swagger.yaml

```diff
@@ -10031,3 +10062,134 @@ definitions:
         type: string
         enum:
         - execute
+
+  #
+  # Vendors API
+  #
+  VendorsAPIJobsListSuccessResponse:
+    allOf:
+    - $ref: '#/definitions/SuccessResponse'
+    - type: object
+      properties:
+        data:
+          type: object
+          required:
+            - lastUpdatedDate
+            - agency
+            - translationJobs
+          properties:
+            lastUpdatedDate:
+              type: string
+              format: date-time
+              description: Date when data for the api response was retrieved.
```

Comment:
API

Reply:
The description doesnt sound right, it's more when the response returned as last calculated.